packages/core/services/AnnotationService/DatabaseAnnotationService/index.ts

```diff
@@ -92,14 +95,13 @@ export default class DatabaseAnnotationService implements AnnotationService {
             .from(this.dataSourceNames);
 
         Object.keys(filtersByAnnotation).forEach((annotationToFilter) => {
-            const annotationValues = filtersByAnnotation[annotationToFilter];
-            if (annotationValues[0] === null) {
+            const appliedFilters = filtersByAnnotation[annotationToFilter];
+            // If an annotation has no filtered values applied, we assume it's a hierarchy or include filter
```

Comment:
Same comment as above, is this still a case since what was previously a sentinel `null` value is now a whole filter as an `IncludeFilter`

Reply:
This condition is currently still fulfilled because of line 74. However, I can change 74 to be an actual Include filter (currently it's a default type one) and do away with this.

Reply:
Oh I see yeah I think keeping it consistent so that the IncludeFilter is used whenever this type of thing happens would be ideal